regression/ansi-c/arch_flags_mcpu_good/test.desc

```diff
@@ -12,12 +12,17 @@ The object file 'object.arm' was compiled from 'source.c' with goto-cc
 along with an ARM cross-compiler on a 64-bit platform with the following
 command line:
 
-  goto-cc --native-compiler=arm-none-eabi-gcc -mcpu=cortex-a15 -c source.c
+  goto-cc -o object.arm --native-compiler=arm-none-eabi-gcc -mcpu=cortex-a15 -c source.c
 
-On Ubuntu, you can get a suitable compiler using:
+To regenerate object.arm on an x86_64 machine, you will need to install
```

Comment:
I'm not sure why these have appeared in this PR, as this is the head commit in the develop branch.

Reply:
I'm not entirely sure why this commit has appeared in the PR, since it is the head commit of the develop branch (or was when I created this branch). I created the branch by checking out develop and then cherry-picking my commits on top of it. 

Nevertheless, I need to rebase against develop again anyway so I will make it disappear then